src/json-formatter.html

```diff
@@ -4,7 +4,10 @@
     <span class="key" ng-if="hasKey"><span class="key-text">{{key}}</span><span class="colon">:</span></span>
     <span class="value">
       <span ng-if="isObject()">
-        <span class="constructor-name">{{getConstructorName(json)}}</span>
+        <span class="json-title" ng-if="!!title" ng-bind="title"></span>
```

Comment:
as the name implied, the getConstructorName function should just return the constructor name.
a new function getDisplayName to wrap the logic will be better.
but the span class is constructor-name, maybe give it another class name